### ISSUE TITLE
[COGNITION]: My VA Benefits screen is reading a lot of data to some assistive devices on content load

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#705]
## Issue
The My VA benefits are wrapped in a div with an `aria-live="polite"` attribute. Some screen readers are reading the alert message and all of the benefits a veteran is signed up for on first load. This is a lot of information to parse, and should be dialed back to just read the important alert messages on load. Screenshot attached below.

## Acceptance Criteria
* As a screen reader user, I want to hear relevant alerts read out (such as homelessness, or if my healthcare application was rejected) when my benefits load.
* I do not want to hear all of the benefits I've signed up for read when the benefits load.

## Environment
* Windows 10
* Firefox, Chrome
* NVDA in particular, when paired with Chrome or Firefox
* https://staging.va.gov/my-va/

## Steps to Recreate
1. Start the NVDA screen reader
2. Log in with a test user who has VA benefits
3. Navigate to the My VA page listed above
4. Verify the H1 is read, then listen for all of the user's benefits to be read out. Should be a very long list of things read before the screen reader stops.

## Possible Fixes (optional)
* I think it would benefit us to add a `role` prop to the `<Alertbox />` [design system component](https://design.va.gov/components/alertboxes). Filing a separate ticket for this work.
* Remove the `aria-live="polite"` attribute from the containing div
* Retest with screen readers for correct handling

## WCAG or Vendor Guidance (optional)
* [Parsing: Understanding SC 4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

## Screenshots or Trace Logs
![Screen Shot 2019-07-26 at 10 10 26 AM (3)](https://user-images.githubusercontent.com/934879/61964448-7cc74c00-af93-11e9-8307-832eecd3b31c.png)

**Comment by alexpappasoddball** [comment]
@Justin-Pickett I believe this one belongs to your team.  Can you confirm and relabel so it gets to your backlog?

**Comment by alexpappasoddball** [comment]
@Justin-Pickett This one has been sitting in the Triage backlog, It seems to belong to your team, can you please move to your backlog.  Let me know if you have any questions.

**Comment by alexpappasoddball** [comment]
Assigning to Justin P. and moving out of Triage Backlog.  Let me know if there are any questions

**Comment by Justin-Pickett** [comment]
Having trouble replicating this issue. Need clearer guidance on what changes are needed for the code. Will coordinate with platform team on trying to get some movement on this issue.

**Comment by 1Copenut** [comment]
@Justin-Pickett Try adding all of the benefits from the select benefits screen and then return to this first view. It should be reading out all of the information, which has been flagged by auditors in other cases.

I'm adding @jenstrickland as a resource to this ticket. She's the VSA accessibility lead and will be able to offer context and guidance.

**Comment by erikphansen** [comment]
Thanks @1Copenut! I think the hangup is the fact that this is an issue using NVDA on Win 10, which we don't have easy access to so we can't replicate it. Is this also an issue using voiceover in Safari?

**Comment by mojaray2k** [comment]
@1Copenut How am I supposed to reproduce this issue? I don't have a windows machine. And that screen reader only has a windows version https://www.nvaccess.org/download/

**Comment by erikphansen** [comment]
@mojaray2k looks like you can replicate it using VoiceOver in Safari based on Trevor's previous comment. Did you give that a shot?

**Comment by mojaray2k** [comment]
@erikphansen Will do that thank you.

**Comment by mojaray2k** [comment]
@1Copenut Can you provide me with a user that has benefits. I think I have the correct attribute removed but just need to test.

**Comment by erikphansen** [comment]
@1Copenut A PR for this has been merged so it can be validated.

**Comment by 1Copenut** [comment]
@erikphansen I spot-tested this with several screen readers, and it's working as expected:
* IE11 + JAWS
* Chrome, Firefox + NVDA
* Safari + VoiceOver

Removing the `aria-live` region was the key. There's another question now about whether we need to add a live region to the homelessness alert, but that's a design question. Do we want to read out "My VA" then pause, then read out the "If you are in danger of becoming homeless" message?

If that's a yes, I can open a new issue. But for the purposes of this one, I will go ahead and close this issue.

/cc @Justin-Pickett
